packages/support/resources/lang/es/actions/dissociate.php

```diff
@@ -32,20 +32,20 @@
 
         'modal' => [
 
-            'heading' => 'Disociar :label seleccionada',
+            'heading' => 'Disociar :label seleccionadas',
```

Comment:
@pathros, should this be `seleccionadas` or `seleccionados` to be consistent with the other translations?

Reply:
Yes, @zepfietje. You're right. Changed "seleccionadas" for "seleccionados" to be consistent.
Also, "disociada" and "disociadas" should be changed to "disociado" and "disociados" to be consistent.

Reply:
Perfect.